operations/scheduler.go

```diff
@@ -515,6 +515,14 @@ func (s *Scheduler) executeOperationPreconditions(ctx context.Context, operation
 		return fmt.Errorf("scheduling for operations of type %s is not allowed", string(types.SUCCEEDED))
 	}
 
+	if time.Now().UTC().After(operation.CreatedAt.Add(s.reconciliationOperationTimeout)) {
+		return &util.HTTPError{
```

Comment:
An http error is returned, but we shouldn't expect this to happen as a result of a REST call, right? It will be a result of the maintainer rescheduling some operation.

Reply:
yes unless at some point its necessary to allow rescheduling operations from a remote system